client/Components/Home.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+/**
+ * conastant function for landing page content
+ * @constant
+ * @returns {void}
+ */
+const Home = () => (
+  <div className="main">
+    <div className="parallax-container">
+      <div className="section no-pad-bot">
+        <div className="container overlay-bg">
+          <br /><br />
+          <h1 className="center teal-text white-text">More Recipes</h1>
+          <div className="row center">
+            <h4 className="col s12 light white-text">
+              More-Recipes provides a platform for users to share awesome and exciting recipe ideas they have invented or learnt...
+            </h4>
+          </div>
+          <br /><br />

```

Comment:
Same with comment above about line breaks

Reply:
I will do that